service/history/wrappers/ratelimited/handler_generated.go

```diff
@@ -109,7 +109,7 @@ func (h *historyHandler) DescribeWorkflowExecution(ctx context.Context, hp1 *typ
 	}
 
 	if !h.allowFunc(hp1.GetDomainUUID(), hp1.Request.GetExecution().GetWorkflowID()) {
-		err = &types.ServiceBusyError{"Too many requests for the workflow ID"}
+		err = &types.UserServiceBusyError{"Too many requests for the workflow ID"}
```

Comment:
Also commented on the idl PR about this. Maybe we should keep one type `ServiceBusyError` and add a `Reason` and `RateLimiterType` fields there. Or something along those lines to indicate error returned due to a specific type of rate limiter. 
In the near future we may introduce more types of rate limits and how/if frontend retries for them is subject to change.